kolibri/core/assets/src/exams/utils.js

```diff
@@ -63,16 +62,14 @@ function convertExamQuestionSourcesV0V2(questionSources, seed, questionIds) {
 
 function convertExamQuestionSourcesV1V2(questionSources) {
   // In case a V1 quiz already has this with the old name, rename it
-  if (every(questionSources, 'counterInExercise')) {
```

Comment:
I am not sure I understand the purpose of changing these functions - is there a bug here we are fixing?

Reply:
When I read this I thought it might miss a case where question sources has _some_ camelCase but not _every_ ... so naturally I overthought it. Pushed a change where it'll just use `some` instead of `every` which seems reasonable unless we're sure it's impossible to have both types in the same sources list.

Reply:
@AlexVelezLl this also does away with the copy / reference issue altogether using the for loop to update things in place